docs/index.md

```diff
@@ -57,7 +92,20 @@ class YourCoreLib(CoreLib):
         self.user = UserDataAccess(db_connection)
         ...
 ```
-
+#### `your_core_lib.yaml` Explained:
+<b>Python script or application that uses a custom CoreLib class and a SqlAlchemyConnectionRegistry class to manage database connections.</b>
```

Comment:
instead of saying `Python script or application that uses a custom` 
just explain the core-lib is a simple stupid python class.  Plain Old Python Object POPO
give link if needed what is POPO[1](https://ink to popo) (try to use and official python link)

something like this
core-lib is a simple POPO (Plain Old Python Object)[1](http://///asdadsad) that is the wrapper/fascade of your core-lib application. 

see if this correct to add:
this is where you are building dependency injection to build our Core-Lib library interface.